pyomo/contrib/doe/__init__.py

```diff
@@ -19,7 +19,8 @@
     "objects that label unknown parameters, experiment inputs, experiment outputs "
     "and measurement error. This avoids string-based naming which is fragile. For "
     "instructions to use the new interface, please see the Pyomo.DoE under the contributed "
-    "packages documentation at `https://pyomo.readthedocs.io/en/latest/contributed_packages/doe/doe.html`"
+    "packages documentation at "
+    #    "`https://pyomo.readthedocs.io/en/latest/contributed_packages/doe/doe.html`"
```

Comment:
@djlaky you should be able to revert this change. The URL checker was failing from a URL unrelated to this PR that was temporarily unavailable but seems to be back online now.